README.md

```diff
@@ -1,2 +1,2 @@
 # domotica
-Project domotica voor groep 5 haagse hogeschool Delft
+Project Domotica voor groep 5 haagse hogeschool Delft.
```

Comment:
shouldnt this be Haagse Hogeschool Delft?

Reply:
Yes it is

Reply:
I've fixed the capitalisation issue.